test/DriverFactoryTest.php

```diff
@@ -90,26 +88,37 @@ public function simplifiedDriverClassProvider() : array
 
     public function testItSupportsSettingDefaultDriverUsingMappingDriverChain() : void
     {
-        $container = $this->prophesize(ContainerInterface::class);
-        $container->has('config')->willReturn(true);
-        $container->get('config')->willReturn([
-            'doctrine' => [
-                'driver' => [
-                    'orm_default' => [
-                        'class' => MappingDriverChain::class,
-                        'default_driver' => 'orm_stub',
-                    ],
-                    'orm_stub' => [
-                        'class' => TestAsset\StubFileDriver::class,
+        $container = $this->createContainerMockWithConfig(
+            [
+                'doctrine' => [
+                    'driver' => [
+                        'orm_default' => [
+                            'class' => MappingDriverChain::class,
+                            'default_driver' => 'orm_stub',
+                        ],
+                        'orm_stub' => [
+                            'class' => TestAsset\StubFileDriver::class,
+                        ],
                     ],
                 ],
             ],
-        ]);
-
-        $factory = new DriverFactory();
+            2
+        );
 
-        $driver = $factory($container->reveal());
+        $driver = (new DriverFactory())->__invoke($container);
         $this->assertInstanceOf(MappingDriverChain::class, $driver);
         $this->assertInstanceOf(TestAsset\StubFileDriver::class, $driver->getDefaultDriver());
     }
+
+    /**
+     * @param array<string, mixed> $config
+     */
+    private function createContainerMockWithConfig(array $config, int $expectedCalls = 1) : ContainerInterface
+    {
+        $container = $this->getMockBuilder(ContainerInterface::class)->getMock();
```

Comment:
`$this->createMock()` suffices

Reply:
Addressed with bbe09841c12999e74a99b7af5d99804871fc90f2